src/OpenTelemetry.AutoInstrumentation/OpenTelemetry.AutoInstrumentation.csproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="..\CommonExcludedAssets.props" />
 
   <PropertyGroup>
-    <Version>0.6.0</Version>
```

Comment:
Did you have chance to verify what is the library version after this change?

Reply:
yes, they are still 0.6.0